Composer/packages/client/src/components/ProjectTree/ProjectTree.tsx

```diff
@@ -224,6 +224,10 @@ export const ProjectTree: React.FC<Props> = ({
 
   const jsonSchemaFilesByProjectId = useRecoilValue(jsonSchemaFilesByProjectIdSelector);
 
+  const createSubtree = useCallback(() => {
```

Comment:
@cwhitten we are now memoizing the project tree creation. This has removed the delay we have when we click on a tree item